.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
+    - os: osx # OSX 10.11
+      osx_image: xcode7.2
+      dotnet: 1.0.0-preview2-003121
+
+script:
```

Comment:
each line is a shell command, ( you can do `- which dotnet` )
travis by default run `dotnet --info` after install so the log always contains the version info
These are just a familiar smoke test, if the use know travis it's normal